src/datatypes/point.c

```diff
@@ -19,3 +19,22 @@ float Point_lon(SIValue point) {
 	return point.point.longitude;
 }
 
+SIValue Point_GetCoordinate(SIValue point, SIValue key) {
+	ASSERT(SI_TYPE(point) == T_POINT);
+	ASSERT(SI_TYPE(key) & T_STRING);
```

Comment:
why not == on both lines?

Reply:
Done in [666387d](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/666387d61e00cc67bae2abd7f23a1fd8803902b6)